src/Luhn/Algorithm/LuhnAlgorithm.php

```diff
@@ -68,10 +68,12 @@ protected function cleanAndSplitInput(string $input): array
         if ($inputLength === 0) {
             throw new InvalidArgumentException;
         }
-
+        
+        $inputLength--;
```

Comment:
Using the `$inputLength--` firstly to avoid duplicated `$inputLength-1` calculating.